src/js/profile/blocks/content_block.js

```diff
@@ -62,10 +81,35 @@ export class ContentBlock extends Component {
         $(chartDescClass, this.container).html(this.description);
     }
 
+    createVersionData() {
+        $('.profile-indicator__version', this.container).remove();
+
+        if (this.indicator.version_data.model.isActive || this.activeVersion === null) {
+            return;
+        }
+
+        let ele = document.createElement('p');
+        $(ele).addClass('profile-indicator__version');
+        $(ele).html(`This indicator applies to 
+        <a href="#">
+            <span class="version-link is--location no-link">${this.geography.name}</span>
+        </a> in the 
+        <a href="#">
+            <span class="version-link is--version no-link">${this.indicator.version_data.model.name}</span>
+        </a>. The map currently shows the 
+        <a href="#">
+            <span class="version-link is--current no-link">${this.activeVersion.model.name}</span>
+        </a>.`);
```

Comment:
we should b cloning these from the component library rather than hardcoding classes.

can fix later